arch/x86/Kconfig

```diff
@@ -333,6 +333,10 @@ config KASAN_SHADOW_OFFSET
 	depends on KASAN
 	default 0xdffffc0000000000
 
+config KFENCE_STATIC_POOL
```

Comment:
Shouldn't it be something like HAVE_KFENCE_STATIC_POOL that is always true on x86?
Or do you want a possibility to test the non-static pool as well?

Reply:
This is an immutable Kconfig option, although going with some HAVE_ option is probably better. I'll change it.